### DISCUSSION TITLE
Question for srmeier

### DISCUSSION THREAD

**Discussion by UTengine** [ko4life-net/ko#205] [Q&A]
I"ve read in your source that you were loading experimentally some v2000 bill/board n3fx and found some ??? Bytes long section after a boolean. Considering from all the schmoozing with game files I've done recently I was asking myself (if) that boolean is true numkey color :) can you confirm?

**Comment by srmeier** [comment]
> I"ve read in your source that you were loading experimentally some v2000 bill/board n3fx and found some ??? Bytes long section after a boolean. Considering from all the schmoozing with game files I've done recently I was asking myself (if) that boolean is true numkey color :) can you confirm?

Humm, it has been a while since I tried loading anything v2000. However, you could be correct, especially if true numkey color isn't always used for billboards.
